midend/global_copyprop.cpp

```diff
@@ -51,7 +51,7 @@ void compareValuesInMaps(std::map<cstring, const IR::Expression*> *oldValues,
     for (auto it : *newValues) {
         auto oldValue = (*oldValues)[it.first];
         if (((it.second == nullptr) ^ (oldValue == nullptr)) ||
-                it.second && oldValue && !(it.second->equiv(*oldValue)))
+            (it.second && oldValue && !(it.second->equiv(*oldValue))))
```

Comment:
Does this fix something else or are these just cosmetic changes?

Reply:
this is a clang warning